sdk/include/opentelemetry/sdk/metrics/state/attributes_hashmap.h

```diff
@@ -71,108 +76,115 @@ class AttributesHashMap
    */
   Aggregation *GetOrSetDefault(const opentelemetry::common::KeyValueIterable &attributes,
                                const AttributesProcessor *attributes_processor,
-                               std::function<std::unique_ptr<Aggregation>()> aggregation_callback,
-                               size_t hash)
+                               std::function<std::unique_ptr<Aggregation>()> aggregation_callback)
   {
-    auto it = hash_map_.find(hash);
+    MetricAttributes attr{attributes, attributes_processor};
```

Comment:
can we add a TODO here for possible optimization - as for now it will copy the attribute list in memory even if it already exist in the hashmap.

Reply:
Added TODO.